src/Draggable/Plugins/AutoScroll/README.md

```diff
@@ -17,16 +17,25 @@ Determines the scroll speed. Default: `10`
 **`sensitivity {Number}`**  
 Determines the sensitivity of scrolling. Default: `30`
 
+**`scrollableElements {HTMLElement[]}`**  
+Allows users to specify their own scrollable elements, rather than letting Draggable compute these automatically. Default: `[]`
+
 ### Examples
 
 ```js
 import {Draggable} from '@shopify/draggable';
 
+const customScrollableElements = document.querySelectorAll('.my-custom-scroll-elements')
+
 const draggable = new Draggable(document.querySelectorAll('ul'), {
   draggable: 'li',
   autoScroll: {
     speed: 6,
     sensitivity: 12,
+    scrollableElements: [
+      ...customScrollableElements,
+      document.documentElement,
```

Comment:
Is this necessary? Don't we already use the `html` node if none of the `customScrollableElements` are found? I might not be understanding this correctly... but thats what I thought this line did:

`return closest(target, this.options.scrollableElements) || document.documentElement;`

Reply:
Good catch and my mistake 👍 Added ` || document.documentElement;` later on and forgot to remove it from the docs. Will remove now and 🚀